components/account/AccountHeroStats.tsx

```diff
@@ -345,17 +353,35 @@ const AccountHeroStats = ({
                 </div>
               ) : null}
             </div>
-            <p className="mb-0.5 mt-1 text-left text-2xl font-bold text-th-fgd-1 lg:text-xl xl:text-2xl">
-              <FormatNumericValue
-                value={accountPnl}
-                decimals={2}
-                isUsd={true}
-              />
-            </p>
-            <div className="flex space-x-1.5">
-              <Change change={rollingDailyPnlChange} prefix="$" size="small" />
-              <p className="text-xs text-th-fgd-4">{t('rolling-change')}</p>
-            </div>
+            {!accountHidden ? (
+              <>
+                <p className="mb-0.5 mt-1 text-left text-2xl font-bold text-th-fgd-1 lg:text-xl xl:text-2xl">
+                  <FormatNumericValue
+                    value={accountPnl}
```

Comment:
Can use accountHidden in the component here so we don't repeat the jsx i.e. value={!accountHidden ? accountPnl : 0}